packages/integration-tests/src/tests/api/interaction/sign-in-with-password-identifier/sad-path.test.ts

```diff
@@ -0,0 +1,212 @@
+import { ConnectorType } from '@logto/connector-kit';
+import { InteractionEvent, SignInMode } from '@logto/schemas';
+
+import { suspendUser } from '#src/api/admin-user.js';
+import { putInteraction } from '#src/api/interaction.js';
+import { updateSignInExperience } from '#src/api/sign-in-experience.js';
+import { initClient } from '#src/helpers/client.js';
+import {
+  clearConnectorsByTypes,
+  setSmsConnector,
+  setEmailConnector,
+} from '#src/helpers/connector.js';
+import { expectRejects } from '#src/helpers/index.js';
+import { enableAllPasswordSignInMethods } from '#src/helpers/sign-in-experience.js';
+import { generateNewUser } from '#src/helpers/user.js';
+import { generateName, generatePassword } from '#src/utils.js';
+
+describe('Sign-in flow sad path using password identifiers', () => {
+  beforeAll(async () => {
+    await enableAllPasswordSignInMethods();
```

Comment:
should set the SIE exclusively under each test case? Feel like `enableAllPasswordSignInMethods` for this test suite is useless.

Reply:
This flow test rely on `enableAllPasswordSignInMethods` sie settings.
Some special tests will setup their own configs and reset accordingly.